### ISSUE TITLE
Feature request: time step increment and disable seconds

### ISSUE THREAD

**Issue by duvillierA** [pytesNET/tail.DateTime#9]
Hi, 

I'm giving a try of your library. 

Some features could be nice for customizing the behavior.

**First request**: Able to modify separately the step increment of Hours, Minutes, Seconds

```
  // example
 <input type="number" name="dt[m]" value="" min="00" max="59" step={{MinuteStep || 5}} />
 <input type="number" name="dt[m]" value="" min="00" max="59" step={{SecondStep || 5 }} />
```

**Second request**: Able to disabled separately time Minutes or Seconds

Many Thanks.

**Comment by SamBrishes** [comment]
Hellow,

thanks for your requests! I'll try to implement your 2 features within the next Update.

Sincerely,
Sam.

**Comment by SamBrishes** [comment]
Hellow,

Your first request is now available in 0.4.3, use the options

```
timeStepHours: 1,
timeStepMinutes: 5,
timeStepSeconds: 5,
```

to change the step increment of the respective time value.

Sincerely,
Sam.

**Comment by SamBrishes** [comment]
Hellow,

the feature has been added, you can disable Hours, Minutes and Seconds by setting the option to false:

```javascript
tail.DateTime(".datetime", {
    timeHours: null,
    timeMinutes: null,
    timeSeconds: 0
});
```

Happy new Year. :tada: 

Sincerely,
Sam.
